domains/snbhowmik.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "A Portfolio site for my Academic Career",
+    "repo": "https://github.com/snbhowmik/portfolio",
+    "owner": {
+        "username": "snbhowmik",
+        "linkedin": "https://www.linkedin.com/in/snbhowmik/",
```

Comment:
You need to add email line, though you can leave it blank if you don't want to give your email.